core/Symbols.cc

```diff
@@ -502,6 +502,15 @@ string Symbol::toStringFullName(const GlobalState &gs) const {
 }
 
 string Symbol::showFullName(const GlobalState &gs) const {
+    if (this->owner == core::Symbols::PackageRegistry()) {
+        // Pretty print package name (only happens when `--stripe-packages` is enabled)
+        auto nameStr = this->name.show(gs);
+        constexpr string_view packageNameSuffix = "_Package"sv;
+        if (absl::EndsWith(nameStr, packageNameSuffix)) {
+            // Foo_Bar_Package => Foo::Bar
+            return absl::StrReplaceAll(nameStr.substr(0, nameStr.size() - packageNameSuffix.size()), {{"_", "::"}});
```

Comment:
Is there a test that exercises this StrReplaceAll functionality? I couldn't find one.

Reply:
Yeah! This is what symbol search (workspace symbols) uses to display symbols to the user.